docs/file.html

```diff
@@ -0,0 +1,62 @@
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>�����-����. ������� ��������� ����������</title>

```

Comment:
Please, put .html response files under tmp/articles directory. They should not be presented in the repository.